pkg/controller/controller.go

```diff
@@ -56,7 +56,35 @@ import (
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 )
 
+type secretParamsType struct {
+	name                         string
+	deprecatedSecretNameKey      string
+	deprecatedSecretNamespaceKey string
+	secretNameKey                string
+	secretNamespaceKey           string
+}
+
 const (
+	// CSI Parameters that are put into fields and are stripped
+	// from Parameters passed to CreateVolume
+	csiParameterPrefix = "csi.storage.k8s.io/"
+
+	prefixedFsTypeKey = csiParameterPrefix + "FSType"
```

Comment:
do we want these keys to be capitalized?

Reply:
@saad-ali what do you think, looks like the secret parameters were always camelcased. Whereas the old FS Type parameter was `fstype`.

Options:
- `fstype`
- `FsType`
- `fsType` [my preference]
- `FSType`
and I guess all the prefixed secret ones should be camel cased and not start with a capital.